include/event2/visibility.h

```diff
@@ -52,4 +52,11 @@
 
 #endif
 
+// For internal symbols referenced by event-extra
+#if defined(event_core_EXPORTS) || (defined(EVENT__NEED_DLLIMPORT) && defined(_MSC_VER) && !defined(EVENT_BUILDING_REGRESS_TEST))
+#define EVENT2_INTERNAL_EXPORT EVENT2_EXPORT_SYMBOL
```

Comment:
I hope it was possible. It's needed to build event_extra.so/.dll.
To be clear, exported symbols are significantly reduced even with this, compared to .so files built by ./configure && make.